src/app/i18n/pt.json

```diff
@@ -195,48 +238,54 @@
     "range": "Alcance",
     "ranget": "Alcance",
     "regen": "Taxa de regeneração",
-    "reload": "Recarregar",
-    "rof": "Taxa de rajada",
+    "reload": "Tempo de recargar,
```

Comment:
You missed a closing quote here.

Reply:
I'll fix them right away.

Reply:
I think I fixed it. I don't really know how to use git to be honest. I need to learn this better.
from "reload": "Tempo de recargar,
to "reload": "Tempo de recarga",